bin/withdrawer.ts

```diff
@@ -0,0 +1,121 @@
+import { Command } from "https://deno.land/x/cliffy@v1.0.0-rc.1/command/mod.ts";
+import { Confirm, Input, prompt, Secret, Select } from "https://deno.land/x/cliffy@v1.0.0-rc.1/prompt/mod.ts";
+import { Ed25519Keypair, RawSigner } from "@mysten/sui.js";
+import { getKeypair } from "../vault.ts";
+import { decodeKeypair } from "../utils.ts";
+import { provider, sendSuiObjects, withdrawStakeObjects } from "../sui.ts";
+
+type Prompt = {
+    provider: "vault" | "plain-text";
+    path: string | undefined;
+    key: string | undefined;
+    keypair: string | undefined;
+};
+
+export type SendPrompt = Prompt & {
+    amount: number;
+    recipient: string;
+};
+
+let withdrawPrompt: Prompt;
+let sendPrompt: SendPrompt;
+
+const getSigner = async (prompt: Prompt | SendPrompt): Promise<RawSigner> => {
```

Comment:
```suggestion
const getSigner = async (prompt: Prompt): Promise<RawSigner> => {
```

`SendPrompt` fields are not used here